client/reader/list-stream/header.jsx

```diff
@@ -38,7 +39,20 @@ const ListStreamHeader = ( {
 			</span>
 
 			<div className="list-stream__header-details">
-				<h1 className="list-stream__header-title">{ title }</h1>
+				<div className="list-stream__header-title">
+					<h1>{ title }</h1>
+					{ ! isPublic && (
+						<div
+							className="list-stream__header-title-privacy"
+							title={ translate( 'Private list' ) }
+						>
+							<Gridicon icon="lock" size={ 18 } />
+							<span className="list-stream__header-title-privacy-label screen-reader-text">
+								{ translate( '(private list)' ) }
```

Comment:
do we need the parens? seems like that would read oddly.

Reply:
I did some [further research](https://css-tricks.com/accessible-svgs/) on this and it seems like adding a `title` to the SVG is the best approach (so no parens needed!). 

Added support for this in the Gridicon component in fee8e22 👍